README.md

```diff
@@ -2590,5 +2590,5 @@ See [LICENSE](./LICENSE) & [NOTICE](./NOTICE) in the source for details, or the
 
 [ghnotice]: https://github.com/mumbleskates/bilrost/blob/bilrost/NOTICE
 
-Copyright 2023-2025 Kent Ross  
+Copyright 2023-2025 Kent Ross
```

Comment:
the trailing double-space here is actually markdown; it forces the newline to be real, otherwise markdown will wrap the lines together into a single paragraph.